### PR TITLE
Shooting Improvement

### PR DESCRIPTION
**Pull request by YoniKiriaty**
Fixed issue with shooting where it would shoot at bad angles and put an ugly fix on an issue with pathplanner where it doesn't move if it's 0.5 meters apart from its target position.

This is temporary until better ways are found or made.

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBPathFindHolonomic.java
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBPathFindHolonomic.java:1]: exactly the same as the wpilib Autobuilder but has a paramater for the translational tolerance

File: src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBAutoBuilder.java
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBAutoBuilder.java:1]: exactly the same as the wpilib Autobuilder but has a paramater for the translational tolerance

File: src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBPathfindingCommand.java
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/utils/GBPathFinding/GBPathfindingCommand.java:1]: exactly the same as the PathPlanner PathFindingCommand but has a paramater for the translational tolerance

File: src/main/java/edu/greenblitz/robotName/Robot.java [hunk 1]
```diff
@@ -85,13 +84,14 @@ public void robotPeriodic() {
...
         NamedCommands.registerCommand("grip", new NoteToShooter().raceWith(new WaitCommand(1)));
-        AutoBuilder.configureHolonomic(
+        GBAutoBuilder.configureHolonomic(
                 SwerveChassis.getInstance()::getRobotPose,
                 SwerveChassis.getInstance()::resetChassisPose,
                 SwerveChassis.getInstance()::getRobotRelativeChassisSpeeds,
                 SwerveChassis.getInstance()::moveByRobotRelativeSpeeds,
                 ChassisConstants.PATH_FOLLOWER_CONFIG,
                 () -> FMSUtils.getAlliance() == DriverStation.Alliance.Red,
+                ChassisConstants.TRANSLATIONAL_TOLERANCE,
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/Robot.java:94]: add one for the rotational as well

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Comment by yonnbro** [comment]
Why do we copy so much code from PP? Why not inherit from the PathPlanning Command and just overiding the initialize function?

**Comment by YoniKiriaty** [comment]
> Why do we copy so much code from PP? Why not inherit from the PathPlanning Command and just overiding the initialize function?

The objects used in the path planning command are private, so you can not access them from inside the overridden initialize.

**Review by bossyshrimp68 — APPROVED** [review]

**Pull request merged**